src/com/franz/sud/components/engine/GameManager.java

```diff
@@ -306,32 +297,22 @@ private void checkForEnemy() {
      * Gets the narrative of the room. Then sets the isNarrated value to true.
      * @param index
      */
-    private void getNarrative(int index) {
+    private void  getNarrative(int index) {
         Room r = map.getHeroLocation();
         Narrative n = gameNarrative.getNarrative(r);
 
-        try {
-            if (n.isNarrated()) {
-                return;
-            }
-        } catch (NullPointerException e) {
+        if (n.isNarrated()) {
             return;
         }
 
-        try {
-            String[] strings = n.getStrings(index);
-            for (String s : strings) {
-                sleep(3);
-                System.out.println(ANSI_BLUE + s);
-            }
-        } catch (NullPointerException e) {
-
+        String[] strings = n.getNarrative(index);
+        for (String s : strings) {
+                sleep(2);
+            System.out.println(ANSI_BLUE + s);
         }
         System.out.println(ANSI_BLACK);
-        sleep(1);
+        sleep(2);
 
-        if (index > 0 ) {
-            n.setNarrated(true);
-        }
+        if (index == MAX_NARRATIVE ) n.setNarrated(true);

```

Comment:
```
if (n.isNarrated()) {
            return;
}
```

```
if (index == MAX_NARRATIVE ) n.setNarrated(true);
```
Be consistent on how you format your code. 

But honestly, I like the first one better.